docs/edgeql/statements/describe.rst

```diff
@@ -0,0 +1,197 @@
+.. _ref_eql_statements_describe:
+
+DESCRIBE
+========
+
+:eql-statement:
+
+``DESCRIBE`` -- provide human-readable description of a schema or a
+schema object
+
+.. eql:synopsis::
+
+    DESCRIBE SCHEMA [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    DESCRIBE <schema-type> <name> [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    # where <schema-type> is one of
+
+      OBJECT
+      ANNOTATION
+      CONSTRAINT
+      FUNCTION
+      LINK
+      MODULE
+      PROPERTY
+      SCALAR TYPE
+      TYPE
+
+Description
+-----------
+
+``DESCRIBE`` generates a human-readable description of a schema object.
+
+The output of a ``DESCRIBE`` command is a :eql:type:`str` , although
+it cannot be used as an expression in queries. It is intended as a
+quick way to look at a piece of the database schema.
```

Comment:
"It is intended as a quick way to look at a piece of the database schema." -- i'd drop this.

Reply:
OK